spec/system/visit_change_password_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe 'ChangePassword', type: :system do
+  before(:each) do
+    login_gds_user
+  end
+
+  it 'shows form' do
+    visit profile_change_password_path
+    expect(page).to have_content t('password.current_password_lbl')
+  end
+
+  it 'shows errors when an empty form is submitted' do
+    visit profile_change_password_path
+    click_button 'Change password'
+    forms = page.find("#new_change_password_form")
+    expect(forms).to have_content "Old password can't be blank"
+    expect(forms).to have_content "Password can't be blank"
+    expect(forms).to have_content "Password confirmation can't be blank"
+  end
+
+  it 'submits the form correctly without error' do 
+    stub_cognito_response(method: :change_password,payload: {})
```

Comment:
Using the stub cognito response here helped stop my pre commit tests from failing but i'm not sure i've used it entirely correctly here, would appreciate some feedback on this 👍

Reply:
yep, this is perfectly fine